flutter_local_notifications/README.md

```diff
@@ -312,139 +487,16 @@ await flutterLocalNotificationsPlugin.cancelAll();
 ```
 
 
-### Get details on if the app was launched via a notification created by this plugin
+### Get details for whether the app was launched via a notification created by this plugin
```

Comment:
Can you revert? This makes it more simpler to read .I don't believe the use of "whether" is correct either. I've not seen usages in sentences like "...on whether or not..." but not "for whether"

Reply:
Reverted